src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyConsoleLogger.cs

```diff
@@ -2,12 +2,50 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Concurrent;
 using System.Text;
 using Microsoft.Extensions.Logging;
 using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.WebAssembly.Services
 {
+    /// <summary>
+    /// A provider of <see cref="WebAssemblyConsoleLogger{T}"/> instances.
+    /// </summary>
+    public class WebAssemblyConsoleLoggerProvider : ILoggerProvider
+    {
+        private ConcurrentDictionary<string, WebAssemblyConsoleLogger<object>> _loggers;
+        private IJSInProcessRuntime _jsRuntime;
```

Comment:
```suggestion
        private readonly IJSInProcessRuntime _jsRuntime;
```

Reply:
This value is modified in the Dispose method so it's not immutable.

Reply:
Saw the thread below. Makes sense. Updated.